readme.md

```diff
@@ -41,8 +47,15 @@ class Application extends BaseApplication
         ]);
 
         // Load the authenticators, you want session first
-        $service->loadAuthenticator('Authentication.Session');
-        $service->loadAuthenticator('Authentication.Form');
+        $service->loadAuthenticator('Authentication.Session', [
+            'sessionKey' => 'Auth'
+        ]);
+        $service->loadAuthenticator('Authentication.Form', [
+            'fields' => [
+                'username' => 'email',
```

Comment:
Please change to default '`'username' => 'username'`. You can add a comment or another example with non default config.